src/main/resources/default/assets/tycho/scripts/charts.js.pasta

```diff
@@ -345,3 +345,19 @@ function barChart(selector, labels, datasets) {
         options: customConfig
     });
 }
+
+function dougnutChart(selector, labels, data) {
+    new Chart($(selector).get(0).getContext("2d"), {
```

Comment:
jQuery? Don't we want to get rid of it in the future? =)

Reply:
yep, also JSdoc is missing (plz compare tochart types above....